rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -4688,11 +4770,25 @@ bool ppu_interpreter::MTFSFI(ppu_thread& ppu, ppu_opcode_t op)
 	}
 	else
 	{
-		u32 i = op.i;
-		ppu.fpscr.bits[bf + 3] = i & 1; i >>= 1;
-		ppu.fpscr.bits[bf + 2] = i & 1; i >>= 1;
-		ppu.fpscr.bits[bf + 1] = i & 1; i >>= 1;
-		ppu.fpscr.bits[bf + 0] = i & 1;
+		// TODO: Constexpr
```

Comment:
Just don't use std::invoke to call lambda, and it should compile as static constexpr.